lang/en.json

```diff
@@ -352,6 +352,34 @@
     "CHAT.ContextualMenu" : "A contextual menu could be available on this card (right click)",
     "CHAT.EditTest" : "Edit Test",
     "CHAT.OpposedTest" : "Opposed Test",
+    "CHAT.CommandLine.Help.Commands" : "Tables,Conditions,CharacterGeneration,NameGeneration,AvailabilityTest,Pay",
+    "CHAT.CommandLine.Help.Label.Command" : "Command",
+    "CHAT.CommandLine.Help.Label.Example": "Example",
+    "CHAT.CommandLine.Help.Label.Note": "Note",
+    "CHAT.CommandLine.Help.Tables.Title" : "Tables",
+    "CHAT.CommandLine.Help.Tables.Usage.Command" : "/table <table-name> <column> <modifier>",
+    "CHAT.CommandLine.Help.Tables.Usage.Example" : "/table critarm +20 /table career human",
+    "CHAT.CommandLine.Help.Tables.Usage.Note" : "/table <anything-unrecognized> will pull up the table help menu",
+    "CHAT.CommandLine.Help.Conditions.Title" : "Conditions",
+    "CHAT.CommandLine.Help.Conditions.Usage.Command" : "/cond <condition-name>",
+    "CHAT.CommandLine.Help.Conditions.Usage.Example" : "/cond ablaze /table unconsc",
```

Comment:
Should be `/cond ablaze` `/cond unconsc`

Reply:
I did the change on the wiki but not in my code :)